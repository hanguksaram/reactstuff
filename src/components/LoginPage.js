import React from 'react'
import Spinner from './Spinner'


const LoginPage = (props) => (
    <div className = 'box-layout'>
        <div className = 'box-layout__box'>
            <Spinner/>
        </div>
    </div>
)


// const mapDispatchToProps = (dispatch) => ({
//     startLogin : () => dispatch(startLogin())
// })

export default LoginPage