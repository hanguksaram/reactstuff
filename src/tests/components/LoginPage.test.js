
import configureMockStore from 'redux-mock-store'
import React from 'react'
import { shallow } from 'enzyme'
import LoginPage from '../../components/LoginPage'
import configureStore from '../../store/configureStore'

const store = configureStore()


test('should render loginpage correctly', () => {
    
    const spy = jest.fn()
    const wrapper = shallow(<LoginPage store = {spy}/>)
    expect(wrapper).toMatchSnapshot()
})



// test('should call startLogin on button click', () => {
   
//     const startLogIn = jest.fn();
//     const wrapper = shallow(<LoginPage store = {store} startLogin = {startLogIn}/>)
//     wrapper.find('button').simulate('click')
//     expect(startLogIn).toHaveBeenCalled()
// })