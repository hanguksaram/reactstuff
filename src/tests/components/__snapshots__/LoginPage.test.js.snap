// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loginpage correctly 1`] = `
<LoginPage
  startLogin={[Function]}
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
/>
`;
